src/StripeTests/Entities/Identity/VerificationReports/VerificationReportTest.cs

```diff
@@ -0,0 +1,41 @@
+namespace StripeTests.Identity
+{
+    using Newtonsoft.Json;
+    using Stripe.Identity;
+    using Xunit;
+
+    public class VerificationReportTest : BaseStripeTest
+    {
+        public VerificationReportTest(StripeMockFixture stripeMockFixture)
+            : base(stripeMockFixture)
+        {
+        }
+
+        [Fact]
+        public void Deserialize()
+        {
+            string json = this.GetFixture("/v1/identity/verification_reports/vs_123");
+            var verificationReport = JsonConvert.DeserializeObject<VerificationReport>(json);
+            Assert.NotNull(verificationReport);
+            Assert.IsType<VerificationReport>(verificationReport);
+            Assert.NotNull(verificationReport.Id);
+            Assert.Equal("identity.verification_report", verificationReport.Object);
+        }
+
+        [Fact]
+        public void DeserializeWithExpansions()
+        {
+            string[] expansions =
+            {
+              "cardholder",
```

Comment:
That test is incorrect, there's no cardholder on VR, why doesn't this error?

Reply:
Great catch. VR has no expansions so I've removed this test. The "not-failing" behavior originates in stripe-mock. I really can't make sense of stripe-mock's behavior here:

`curl localhost:12111/v1/identity/verification_sessions?expand[]=customer -u sk_test_xzy:` doesn't throw a validation error, but `curl localhost:12111/v1/identity/verification_sessions?expand[]=foo -u sk_test_xzy:` throws a 500.

In any case, the test is fixed but the underlying issue is not.